packages/pulpcore/pulpcore-selinux/pulpcore-selinux.spec

```diff
@@ -0,0 +1,81 @@
+%define selinux_variants mls strict targeted
+%define selinux_modules pulpcore_port pulpcore
+
+Name:           pulpcore-selinux
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        SELinux policy for Pulp 3
+
+License:        GPL2+
+URL:            https://pulpproject.org
+Source0:        https://codeload.github.com/pulp/%{name}/tar.gz/%{version}#/%{name}-%{version}.tar.gz
+
+BuildRequires:  checkpolicy
+BuildRequires:  selinux-policy-devel
+Requires:       selinux-policy >= %{_selinux_policy_version}
+Requires:       python3-pulpcore
```

Comment:
I am still unsure whether pulpcore-selinux should require python3-pulpcore or the other way round :woman_shrugging:

Reply:
You only want to ensure when the policy is installed it installs also the base package in most cases. If someone has selinux disabled you do not want to install the policy package. So pulpcore-selinux should require python3-pulpcore and not the other way round.

Does RPM in EL7 already support the rich rule syntax to get the conditional dependency? I am not sure, but I think no. If yes, this could also solve the problem in a more elegant way.

Reply:
No, rich deps are not a thing in EL7, sadly :(